.pipelines/templates/e2e-tests.yml

```diff
@@ -29,7 +29,7 @@ steps:
           python -m pip install --upgrade pip
           pip install -r requirements.txt
           pip install pytest-azurepipelines
-          python -m spacy download en_core_web_sm
+          python -m spacy download en_core_web_lg
```

Comment:
the usage of th large package is on purpuse?

Reply:
Yes, to simulate the real instance